jvm-packages/dev/Dockerfile

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (c) 2019 by Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+FROM centos:6
+
+# Install all basic requirements
+RUN \
```

Comment:
after taking a second look, I think it might be better to keep a separate copy of Dockerfile here to ensure that we have a consistent release env